tck/src/main/java/com/github/bsideup/liiklus/positions/PositionsStorageTests.java

```diff
@@ -0,0 +1,117 @@
+package com.github.bsideup.liiklus.positions;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import reactor.core.publisher.Flux;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public interface PositionsStorageTests extends PositionsStorageTestSupport {
+
+    @Test
+    @DisplayName("Should save new position")
+    default void shouldSavePosition() {
+        GroupId groupId = GroupId.ofString(UUID.randomUUID().toString());
+
+        Map<Integer, Long> positions = asDeferMono(() -> getStorage().update(getTopic(), groupId, 2, 2))
```

Comment:
what if instead of `asDeferMono` we use `await(getStorage().update(getTopic(), groupId, 2, 2))`, where `await` is something like:
```
<T> T await(CompletionStage<T> future) {
    return future.toCompetableFuture().get(10, SECONDS);
}
```
?